Add option to set delimeter for environment variables

leanazulyoro opened the pull request:
closes #14 

This is my first golang code ever, so please check it carefully.
The intention is to have the default delimiter to be the one that was defined already (":|:"), and if the environment_delimiter settings is defined, then use that one instead.

oliver006 commented on main.go:
```diff
@@ -300,7 +301,11 @@ func CreateExecutionPlan(cfg *Config) (Plan, error) {
 				}
 
-				envStr := "^:|:^" + strings.Join(e, ":|:")
+				envsDelimiter := ":|:"
```
Can you make this a const called `defaultEnvVarDelimiter` or similar and move up to the block near line 55?

oliver006 commented on main.go:
```diff
@@ -300,7 +301,11 @@ func CreateExecutionPlan(cfg *Config) (Plan, error) {
 
-				envStr := "^:|:^" + strings.Join(e, ":|:")
+				envsDelimiter := ":|:"
+				if f.EnvironmentDelimiter != "" {
```
How about checking and setting the delimiter up here: https://github.com/oliver006/drone-gcf/blob/58f0568a73bdd522a0d5d10cac36b51c501af61a/main.go#L137

and then just use it here?

leanazulyoro commented:
@oliver006 I've adjusted according to your comments, but the pipeline is not running correctly. I've ran tests locally and they pass. Do you think this is a pipeline issue or it has something to do with the changes I've introduced?

oliver006 commented:
Looks like the go fmt check fails - can you make sure all the files are formatted correctly?

leanazulyoro commented:
@oliver006 great thanks!, I formatted it with `go fmt`. Let me know if there's anything missing.

oliver006 commented:
Released as https://github.com/oliver006/drone-gcf/releases/tag/v1.9.0
